ee/app/livechat-enterprise/server/methods/resumeOnHold.ts

```diff
@@ -1,9 +1,35 @@
 import { Meteor } from 'meteor/meteor';
+import { TAPi18n } from 'meteor/rocketchat:tap-i18n';
 
-import { LivechatRooms, LivechatInquiry } from '../../../../../app/models/server';
+import { LivechatRooms, LivechatInquiry, Messages, Users, LivechatVisitors } from '../../../../../app/models/server';
 import { RoutingManager } from '../../../../../app/livechat/server/lib/RoutingManager';
 import { callbacks } from '../../../../../app/callbacks/server';
 
+const resolveOnHoldCommentInfo = (options: { clientAction: boolean }, room: any): { comment: string; onHoldChatResumedBy: any} => {
```

Comment:
Why are you retrieving the user who performed the action inside the method rather than passing this information as a parameter?